python/lsst/obs/subaru/strayLight/yStrayLight.py

```diff
@@ -175,20 +188,19 @@ def evaluate(self, angle_start: Angle, angle_end: Optional[Angle] = None):
         ccd_img : `numpy.ndarray`
             Background data for this exposure.
         """
-        hdulist = fits.open(self._filename)
-        header = hdulist[0].header
+        header = self.getMetadata().toDict()
```

Comment:
If `getMetadata()` returns a PropertyList is the `toDict()` really necessary? Doesn't `header["F_NAXIS2"]` already work fine? or are there some multi-value headers in play?